stopCallback is triggered twice on error

## Context

* Telegraf.js Version: 3.38.0
* Node.js Version: v12.16.0
* Operating System: Ubuntu 18.04.4 LTS

## Expected Behavior

stopCallback is triggered just once as in v.3.32.0

## Current Behavior

stopCallback is triggered twice

## Failure Information (for bugs)

Problem is caused by this lines;
https://github.com/telegraf/telegraf/blob/1273631bd17e22d144526bca213f64ea347f4dec/telegraf.js#L176
https://github.com/telegraf/telegraf/blob/1273631bd17e22d144526bca213f64ea347f4dec/telegraf.js#L197

if error is catched (e.g. 409) this.polling.stopCallback() is executed twice.

v.3.32.0 hasn't got such problem:
https://github.com/telegraf/telegraf/blob/2be9f7a10d1b67546121bccdb567bc26e2d757f8/telegraf.js#L158

### Steps to Reproduce

1. start polling with token that is already in use
2. receive 409 error
3. stopCallback is triggered twice

### Failure Logs

```
[2020.05.21 04:56:33.766] [ERROR]  Failed to process updates. TelegramError: 409: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
    at /root/telebot.im/node_modules/telegraf/core/network/client.js:281:17
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 409,
  response: {
    ok: false,
    error_code: 409,
    description: 'Conflict: terminated by other getUpdates request; make sure that only one bot instance is running'
  },
  description: 'Conflict: terminated by other getUpdates request; make sure that only one bot instance is running',
  parameters: {},
  on: { method: 'getUpdates?offset=0&limit=100&timeout=30', payload: {} }
}
[2020.05.21 04:56:33.770] [ERROR] Callback 1
[2020.05.21 04:56:33.770] [ERROR] Callback 2
```

--- a/telegraf.js
+++ b/telegraf.js
@@ -190,7 +190,6 @@ class Telegraf extends Composer {
         console.error('Failed to process updates.', err)
         this.polling.started = false
         this.polling.offset = 0
-        this.polling.stopCallback && this.polling.stopCallback()
         return []
       })
       .then((updates) => {

